session/xmlrpc.go

```diff
@@ -55,13 +55,10 @@ func (mrt debugRoundTripper) RoundTrip(request *http.Request) (*http.Response, e
 }
 
 // XML-RPC Transport
-type XmlRpcTransport struct {
-	Timeout time.Duration
-}
+type XmlRpcTransport struct{}
 
-const DefaultXmlRpcTimeout = time.Second * 30
+const DefaultXmlRpcTimeout = time.Second * 60
```

Comment:
Are we intentionally allowing transport handlers to have different default timeouts?